doc-content/enterprise-only/processes/interacting-with-processes-claiming-tasks-in-bulk-proc.adoc

```diff
@@ -0,0 +1,12 @@
+[id='interacting-with-processes-claiming-tasks-in-bulk-proc']
+= Claiming tasks in bulk
+
+After tasks are created in {CENTRAL}, you can claim the released tasks in bulk.
```

Comment:
@kaldesai I think it's better talk about available tasks instead of released tasks.

Reply:
@nmirasch, Ack